# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../netkit-base/no-rpc.patch
# Copyright (C) 2020 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- netkit-base-0.17/Makefile.vanilla	2020-08-18 12:42:11.048559228 +0200
+++ netkit-base-0.17/Makefile	2020-08-18 12:42:26.264559146 +0200
@@ -3,7 +3,7 @@
 
 include MCONFIG
 
-SUB = inetd ping
+SUB = ping # inetd
 
 %.build:
 	(cd $(patsubst %.build, %, $@) && $(MAKE))
